pkgs/development/python-modules/pymysql/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, cryptography
+, unittest2
+}:
+
+buildPythonPackage rec {
+  pname = "PyMySQL";
+  version = "0.9.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0gvi63f1zq1bbd30x28kqyx351hal1yc323ckp0mihainb5n1iwy";
+  };
+
+  propagatedBuildInputs = [ cryptography ];
+
+  buildInputs = [ unittest2 ];
+
+  # Wants to connect to MySQL
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Pure Python MySQL Client";
+    homepage = https://github.com/PyMySQL/PyMySQL;
+    license = licenses.mit;
+    maintainers = [ maintainers.fridh maintainers.kalbasit ];
```

Comment:
Why did you add fridh?

Reply:
I ran `git blame` and tracked down the first commit to him, should I remove him?

Reply:
Yes, he's the main python maintainer but probably does not particularly maintain this package.